Makefile

```diff
@@ -82,12 +83,20 @@ op_mode_definitions: $(op_xml_obj)
 	# could mask help strings or mandatory priority statements
 	find $(OP_TMPL_DIR) -name node.def -type f -empty -exec false {} + || sh -c 'echo "There are empty node.def files! Check your interface definitions." && exit 1'
 
+.PHONY: vycvalidators
+vycvalidators:
+	@for module in `ls $(VALIDATORS_C_DIR)`
+	do
+		# Assuming each module is a C file, adjust if necessary for directories
+		$(CC) -o src/validators/$$module $(VALIDATORS_C_DIR)/$$module/$$module.c
```

Comment:
You should also optimize the code using: `-O3`

Reply:
> You should also optimize the code using: `-O3`

Done!